device.go

```diff
@@ -218,12 +218,15 @@ func virtioBlkCCWDeviceHandler(ctx context.Context, device pb.Device, spec *pb.S
 // Here, bridgeAddr is the address at which the brige is attached on the root bus,
 // while deviceAddr is the address at which the device is attached on the bridge.
 func virtioBlkDeviceHandler(_ context.Context, device pb.Device, spec *pb.Spec, s *sandbox) error {
-	// Get the device node path based on the PCI device address
-	devPath, err := getPCIDeviceName(s, device.Id)
-	if err != nil {
-		return err
+	// Allow to use the "VmPath" field set from the input "device"
+	if device.VmPath == "" {
+		// Get the device node path based on the PCI device address
```

Comment:
You may want to update comment here, since the id here is not PCI device address.